RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py

```diff
@@ -46,19 +77,22 @@
     Pt010_Tight    = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
     Pt1020_Tight   = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
     Pt2030_Tight   = cms.vdouble( 0.26, -0.34, -0.24, -0.26),
-    Pt3050_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
+    Pt3040_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
+    Pt4050_Tight   = cms.vdouble( 0.62, -0.21, -0.07, -0.03),
```

Comment:
are these WPs actually used?
I'm curious when is the opportunity to clean up this file.

Reply:
sorry you mean the 80X WPs or in general the Pt3040_* and Pt4050_*?

Reply:
@slava77 the 80X WPs are indeed obsolete.

Reply:
I meant 80X, 53X and alike.

Reply:
However, we need this PR ASAP. We will make sure to clean up this file on the next iteration.